.github/workflows/ci.yml

```diff
@@ -13,12 +13,12 @@ on:
 jobs:
   Ubuntu-Cpp-C:
     runs-on: ubuntu-latest
-    name: "Ubuntu C++ & C (Compiler: ${{ matrix.compiler }}, MPI: ${{ matrix.mpi }})"
+    name: "Ubuntu C++ & C (${{ matrix.compiler }} / ${{ matrix.mpi }})"
     strategy:
       fail-fast: false
       matrix:
         compiler: [GCC, CLANG, ICC]
-        mpi: [OpenMPI, IntelMPI]
+        mpi: [OpenMPI] #, IntelMPI]
```

Comment:
What happened to the IntelMPI?

Reply:
I am having troubles with it, also locally
Since this is not urgent now I wanted to first add the MPI-testing it without and enable it later

overall the Intel compiler gives lots of troubles :/